qa/tasks/cephadm.py

```diff
@@ -981,105 +985,110 @@ def task(ctx, config):
     if 'cluster' not in config:
         config['cluster'] = 'ceph'
     cluster_name = config['cluster']
-    ctx.ceph[cluster_name] = argparse.Namespace()
-
-    ctx.ceph[cluster_name].thrashers = []
-    # fixme: setup watchdog, ala ceph.py
-
-    ctx.ceph[cluster_name].roleless = False  # see below
+    # Support already bootstrapped clusters
+    if not hasattr(ctx.ceph[cluster_name], 'bootstrapped'):
```

Comment:
can we move all of the code below here into a dedicated method?

Reply:
Done